src/depends/libDatabase/LevelDB.cpp

```diff
@@ -88,7 +88,7 @@ LevelDB::LevelDB(const std::string & dbName, const std::string& subdirectory, bo
     // that might not be the current directory (case when 'diagnostic' is true).
     // Its default value is false, and the non-diagnostic path is preserved
     // from the original code.
-    string db_path = diagnostic ? (m_subdirectory + PERSISTENCE_PATH) : ("./" + PERSISTENCE_PATH + (m_subdirectory.empty() ? "" : "/" + m_subdirectory));
+    string db_path = diagnostic ? (m_subdirectory + PERSISTENCE_PATH) : (STORAGE_PATH + "/" + PERSISTENCE_PATH + (m_subdirectory.empty() ? "" : "/" + m_subdirectory));
```

Comment:
I believe this line has an extra `/` in it when resolving the path. `(STORAGE_PATH + "/" + PERSISTENCE_PATH` should be changed to `(STORAGE_PATH + PERSISTENCE_PATH` as far as I can tell. Please check and test that.